djconnectwise/tests/test_commands.py

```diff
@@ -555,10 +568,9 @@ def _test_sync(self, full_option=False):
 
         return out.getvalue().strip()
 
-    def test_sync(self):
-        """Test sync all objects command."""
+    def assert_sync_results(self):
+        output = self.call_sync_command()
```

Comment:
Good point. I've renamed and cleaned this up a bit further. They now run independently of each other.

Reply:
Ok, I've looked at it some more and I think perhaps the original call to `self.test_sync` was supposed to be a call to `self._test_sync`.

In test_full_sync on line 652 see that it is saving the counts of objects before a full sync. I tested it out, and if we don't run a partial sync first all the counts are zero. This means the test is definitely expecting there to be a partial sync run  before the full sync, because we wouldn't need to save the first counts of all the objects if we knew them all to be zero, we would just check for a non-zero value.

I think the best thing to do is to put a call to `self.run_sync_command()` in place of where `self.test_sync` was being called in `test_full_sync`.

Reply:
Ok, thanks for clearing that up. I did not see that. I'll add in that call to `self.run_sync_command()`